src/components/ChatWindow.js

```diff
@@ -21,7 +21,7 @@ class ChatWindow extends Component {
       let messageList = this.props.messageList || [];
       let classList = [
         "sc-chat-window",
-        (this.props.active ? " active" : " inactive")
+        (this.props.isOpen ? " opened" : " closed")
```

Comment:
We actually don't need the spaces because they're joined with one below. Can you remove them plz?